### PR TITLE
fix: Update compile sdk and other library

### PR DESCRIPTION
**Pull request by ak8527**
### Description
Update compile sdk and other library version in build.gradle file.

### CODE REVIEW

**Review by jbeorse — APPROVED** [review]

**Review by linl33 — CHANGES_REQUESTED** [review]
Android Support Library 28.1.1 does not exist. The latest version is 28.0.0.

**Comment by ak8527** [comment]
@linl33 i made the suggested changes.

File: app/build.gradle [hunk 1]
```diff
@@ -20,24 +20,24 @@ android {
...
-    implementation 'com.google.firebase:firebase-core:16.0.1'
-    implementation 'com.google.firebase:firebase-messaging:17.0.0'
-    implementation 'com.google.firebase:firebase-database:16.0.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support:design:28.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'com.google.firebase:firebase-core:16.0.7'
+    implementation 'com.google.firebase:firebase-messaging:17.4.0'
+    implementation 'com.google.firebase:firebase-database:16.1.0'
     implementation 'com.firebase:firebase-jobdispatcher:0.8.5'
```
Comment (linl33) [inline app/build.gradle]: The latest version is 0.8.6. Is there a reason why that version wasn't used?

Comment (linl33) [inline app/build.gradle]: The latest version is 0.8.6. Is there a reason why that version wasn't used?

Comment (ak8527) [reply app/build.gradle]: There is no update available for jobdispatcher at that time, so i don't update it. But now i update it to 0.8.6

**Pull request merged**